java/manager-build.xml

```diff
@@ -298,135 +302,156 @@
 
   <target name="deploy-restart" depends="deploy, restart-tomcat, restart-taskomatic" />
 
-  <target name="is-kubectl-installed">
-    <exec failifexecutionfails="false" resultproperty="kubectl.installed" outputproperty="deploy.pod" executable="kubectl">
-        <arg line="get" />
-        <arg line="pod" />
-        <arg line="-n" />
-        <arg line="${deploy.namespace}" />
-        <arg line="-o"/>
-        <arg line="jsonpath={.items[0].metadata.name}" />
-        <arg line="-lapp=uyuni" />
+  <!-- podman container related -->
+
+  <target name="check-ssh-key" description="Asserts ssh.key is provided and file exists">
+    <!-- Check if ssh.key property is provided -->
+    <fail message="You must provide a ssh.key like -Dssh.key='&lt;ssh-key-full-path&gt;'">
+      <condition>
+        <or>
+          <not>
+            <isset property="ssh.key"/>
+          </not>
+          <equals arg1="${ssh.key}" arg2=""/>
+        </or>
+      </condition>
+    </fail>
+
+    <!-- Check if the provided key file path exists -->
+    <fail message="The specified ssh.key file does not exist">
+      <condition>
+        <not>
+          <available file="${ssh.key}"/>
+        </not>
+      </condition>
+    </fail>
+  </target>
+
+  <target name="is-podman-installed" description="Cheks if podman is installed">
```

Comment:
We don't care if `podman` or `kubectl` is installed: `uyunictl` checks for us, and even checks for `podman-remote`.